packages/ui/src/components/tabs.tsx

```diff
@@ -9,7 +9,10 @@ const tabsListVariants = cva('inline-flex items-center text-foreground-4', {
     variant: {
       default: 'h-9 justify-center rounded-lg bg-muted p-1',
       underline: 'h-11 justify-center gap-4',
-      navigation: 'h-[44px] w-full justify-start gap-6 border-b border-borders-5 px-5',
+      /**
+       * @deprecated Use TabNav component instead
+       */
+      navigation: '',
```

Comment:
Is this variant used anywhere? If so, can we transition those instances and remove this? If not, we can go ahead and kill this off anyway.
```suggestion
```

Reply:
@knagurski the `Tabs` component with the `"navigation"` variant is used in these components from the `views package`: `SandboxPullRequestCompare`, `SandboxExecutions`, `RepoExecutionLayout`, `SettingsAccountPage`, `SettingsProjectNav`. None of these components seem to be imported anywhere in the repo, but I'm unsure if I can remove or even change them (by changing the Tabs variant, for instance).
The goal of this PR is to make particular and obvious changes.

Reply:
I've reverted the styles, and added a `TODO` that we revisit this issue in a separate PR for this, that way we'll be sure these changes don't break anything where it's used